Helper/Data.php

```diff
@@ -258,4 +258,13 @@ public function logError($exception)
             $this->log($exception);
         }
     }
+    
+    public function buildBackendParams($storeId) {
+        return $backendParams = [
```

Comment:
why are are you assigning here?

Reply:
Fixed. Returning array with data instead of variable.